### PR TITLE
Dev branch

### PR DESCRIPTION
**Pull request by TemaTerbi**
BIG UPDATE!

- Integrate swiftUI + UiKit
- Create pickerView
- Fix API Services
- Add new Functions to API Services
- Create Charts (new)
- Connect API with pickerView
- Connect API with Charts
- Dynamic update api info in charts and mainscreen detail info about country

### CODE REVIEW

File: CovidApp/Assets.xcassets/gray.colorset/Contents.json [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+        }
+      },
+      "idiom" : "universal"
+    }
+  ],
+  "info" : {
+    "author" : "xcode",
+    "version" : 1
+  }
+}
```
Comment (userNameIsBoris) [inline CovidApp/Assets.xcassets/gray.colorset/Contents.json:33]: Consider using system semantic colors.

File: CovidApp/Services/DataService.swift [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    var items: [Item] = []
+    
+    func updateData() {
+        testValue = arrayCases.prefix(12)
+    }
+    
+    func getValue() {
+        var dayCounter = 0
+        if testbool {
+            for el in testValue {
```
Comment (userNameIsBoris) [inline CovidApp/Services/DataService.swift:33]: El… Camino?

File: CovidApp/Services/DataService.swift [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+}
+
+final class DataService {
+    static let shared = DataService()
+    
+    var arrayCountries = [Countries]()
+    var monthCases: Int = 0
+    var arrayCases = [Int?]()
+    var testValue: Array<Int?>.SubSequence = []
+    var testbool = true
```
Comment (userNameIsBoris) [inline CovidApp/Services/DataService.swift:23]: Bad naming.

**Review by userNameIsBoris — APPROVED** [review]

File: CovidApp/API/Models/Country.swift [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+struct CountryElement: Codable {
+    let cases: Int?
+
+    enum CodingKeys: String, CodingKey {
+        case cases = "Cases"
+    }
+}
+
+
+typealias byCountry = [CountryElement]
```
[the remark this answers is not in the document]
Comment (TemaTerbi) [reply CovidApp/API/Models/Country.swift:20]: yes, i use this in ApiManager

File: CovidApp/Assets.xcassets/gray.colorset/Contents.json (hunk 1 above)
Comment (TemaTerbi) [reply CovidApp/Assets.xcassets/gray.colorset/Contents.json:33]: UIColor do not support in swiftui

File: CovidApp/Assets.xcassets/stat.imageset/Contents.json [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+    {
+      "idiom" : "universal",
+      "scale" : "3x"
+    }
+  ],
+  "info" : {
+    "author" : "xcode",
+    "version" : 1
+  }
+}
```
[the remark this answers is not in the document]
Comment (TemaTerbi) [reply CovidApp/Assets.xcassets/stat.imageset/Contents.json:21]: I need learning this...

File: CovidApp/Services/DataService.swift [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+    let values: Double
+}
+
+final class DataService {
+    static let shared = DataService()
+    
+    var arrayCountries = [Countries]()
+    var monthCases: Int = 0
+    var arrayCases = [Int?]()
+    var testValue: Array<Int?>.SubSequence = []
```
[the remark this answers is not in the document]
Comment (TemaTerbi) [reply CovidApp/Services/DataService.swift:22]: In api response  [Int?] type

**Pull request merged**